backend/src/main/java/de/bund/digitalservice/ris/norms/application/service/CreateNewWorkService.java

```diff
@@ -0,0 +1,171 @@
+        dokument.getMeta().getFRBRManifestation().setEli(dokumentManifestationEli);
+        dokument.getMeta().getFRBRManifestation().setURI(dokumentManifestationEli.toUri());
+        dokument.getMeta().getFRBRExpression().setEli(dokumentManifestationEli.asExpressionEli());
+        dokument
+          .getMeta()
+          .getFRBRExpression()
+          .setURI(dokumentManifestationEli.asExpressionEli().toUri());
+        dokument
+          .getMeta()
+          .getFRBRExpression()
+          .setFBRDate(newPointIntTime.toString(), "verkuendung");
+        dokument.getMeta().getFRBRWork().setEli(dokumentManifestationEli.asWorkEli());
+        dokument.getMeta().getFRBRWork().setURI(dokumentManifestationEli.asWorkEli().toUri());
+        dokument.getMeta().getFRBRWork().setFRBRnumber(newNaturalIdentifier);
+        dokument.getMeta().getFRBRWork().setFRBRsubtype(newSubtype);
+        // New GUID for übergreifende-id and aktuelle-version-id
+        dokument.setGuid(UUID.randomUUID());
+        dokument.setUebergreifendeGuid(UUID.randomUUID());
+        // Remove just in case previous and next GUIDs
+        dokument.getMeta().getFRBRExpression().deleteAliasPreviousVersionId();
+        dokument.getMeta().getFRBRExpression().deleteAliasNextVersionId();
+      });
+    // set regtxt:form to stammform
+    final RahmenMetadata rahmenMetadata = newNorm.getRahmenMetadata();
+    rahmenMetadata.setForm("stammform");
+
+    var rechtsetzungsdokument = createRechtsetzungsdokumentFromRegelungstext(
+      new Regelungstext(newNorm.getRegelungstext1()),
+      rechtsetzungsdokumentVerkuendung
+    );
+    newNorm.getDokumente().add(rechtsetzungsdokument);
+
+    return newNorm;
+  }
+
+  private Rechtsetzungsdokument createRechtsetzungsdokumentFromRegelungstext(
+    final Regelungstext regelungstext,
+    final Rechtsetzungsdokument rechtsetzungsdokument
+  ) {
+    // Replace <akn:act> element with <akn:documentCollection>
+    DokumentManifestationEli dokumentManifestationEli = regelungstext.getManifestationEli();
+    Meta oldMeta = regelungstext.getMeta();
+    Document doc = regelungstext.getDocument();
+    var actElement = NodeParser.getMandatoryElementFromExpression(".//act", doc);
+    actElement.getParentNode().removeChild(actElement);
+    Element documentCollection = NodeCreator.createElement(
+      Namespace.INHALTSDATEN,
+      "documentCollection",
+      doc.getDocumentElement()
+    );
+    documentCollection.setAttribute(
+      "name",
+      "/akn/ontology/de/concept/documenttype/bund/rechtsetzungsdokument"
+    );
+
+    // Copy meta but only identification and proprietary
+    Element newMetaElement = NodeCreator.createElementWithEidAndGuid(
+      Namespace.INHALTSDATEN,
+      "meta",
+      documentCollection
+    );
+    final NodeList nodes = oldMeta.getElement().getChildNodes();
+    for (int i = 0; i < nodes.getLength(); i++) {
+      final Node node = nodes.item(i);
```

Comment:
We have `NodeParser#nodeListToList` that could simplify this kind of code a bit. But also no need to use it :)